test/user.test.js

```diff
@@ -991,7 +976,19 @@ describe('User', function() {
           if (err) return done(err);
 
           var errorResponse = res.body.error;
-          assert.equal(errorResponse.code, 'LOGIN_FAILED_EMAIL_NOT_VERIFIED');
+
+          // extracting code and details error response
+          let errorExcerpts = {
```

Comment:
plus: errorResponse is not a real error object, Object.assign() should not be required either